lib/mpool/src/mpool_file.c

```diff
@@ -61,6 +61,10 @@ mpool_file_open(
     if (create)
         flags |= (O_CREAT | O_EXCL);
 
+    /* Drop ODIRECT if the mclass doesn't support direct-io */
+    if ((flags & O_DIRECT) && !mclass_directio(mc))
+        flags &= ~O_DIRECT;
+
```

Comment:
s/ODIRECT/O_DIRECT/

Reply:
Greg, please refresh. I fixed this in the last update almost an hour back..